metaprotocol/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema":"../assetlist.schema.json",
+    "chain_name":"LumenX",
```

Comment:
With a chain name of LumenX, why is the folder called metaprotocol? should it not be called 'lumenx'? Seems a tad confusing--wish to avoid if possible

Reply:
Thanks for the kind review. I will make 'lumenx' folder again. You can close this pull request.